docs/release-channels.md

```diff
@@ -0,0 +1,53 @@
+# Release Channels and Upgrades
+
+Microk8s is a snap deploying Kubernetes. Upstream Kubernetes ships a new releases every about three months, while old releases get periodic updates. At the time of this writing the latest release series is `v1.12` with `v1.12.0` being the latest release. On the `v1.13` series, `v1.11.3` is the latest release. It is important to remember that upstream Kubernetes is committed to maintain backwards compatibility only within a release series. That means that your kubernetes will not break while you upgrade from `v1.11.x` to `v1.11.y` but may break if you upgrade from `v1.11.x` to `v1.12.z`.
```

Comment:
> every about three months

about every three months

Reply:
I kinda think this entire paragraph should be removed. The first sentence is redundant. The second through fourth sentences are obvious - I'm not sure why we need them. And the last two sentences are not true.